### PR TITLE
Changed recommended way of creating a prettier config file

### PR DESCRIPTION
**Pull request by danielturus**
## Description

This PR changes the recommended way to create a Prettier config file. 
We should use the power of autocomplete in current IDEs and this is made fairly easy in JavaScript.
Other tools / frameworks use this as well, one example would be [TailwindCSS](https://tailwindcss.com/docs/installation)

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -20,13 +20,15 @@ yarn add --dev --exact prettier
...
+/** @type {import('prettier').Config} */
+module.exports = {
 
-<!-- Note: `echo "{}" > .prettierrc.json` would result in `"{}"<SPACE>` on Windows. The below version works in cmd.exe, bash, zsh, fish. -->
-
-```bash
-echo {}> .prettierrc.json
+}
 ```
+This way, the editor will autocomplete the options you have available according to your installed version of Prettier.
```
Comment (fisker) [inline docs/install.md:30]: Shouldn't editor also do autocomplete to json file too? https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/prettierrc.json

Comment (kachkaev) [reply docs/install.md:30]: JSON files don’t allow `require("some-prettier-plugin")` and not using `require` does not work in Yarn PnP. On the other hand `/** @type {import('prettier').Config} */`, only works if [`@types/prettier`](https://www.npmjs.com/package/@types/prettier) is installed.

Comment (danielturus) [reply docs/install.md:30]: > On the other hand `/** @type {import('prettier').Config} */`, only works if [`@types/prettier`](https://www.npmjs.com/package/@types/prettier) is installed.

Great observation, maybe this should be added in the docs also?

Comment (fisker) [reply docs/install.md:30]: image
image

Comment (karlhorky) [reply docs/install.md:30]: This only works with JSON files. With `.prettierrc.js`, `prettier.config.cjs` etc this does not show errors:

Screenshot 2022-12-08 at 13 23 48

Opened a followup PR for this:

**Pull request closed without merging**